LICENSE

```diff
@@ -186,7 +186,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright (c) Splunk Community, 2024
+   Copyright (c) 2016-2024 Splunk Inc.
```

Comment:
I think we'd just make the copyrights 2024 and not 2016-2024

Reply:
Changed all the copyright dates